dcs-packaging-tool/dcs-packaging-tool-gui/src/main/resources/org/dataconservancy/config/applicationContext.xml

```diff
@@ -135,7 +137,21 @@
     <property name="localCountryCode" value="${localCountryCode}"/>
   </bean>
 
+  <bean id="emailValidator" class="org.dataconservancy.packaging.gui.util.EmailValidator" />
```

Comment:
Not currently, but i'll capture these notes and put them in appropriate cards, or make new cards for them.